packages/Webkul/Admin/src/Resources/views/components/shimmer/notifications/index.blade.php

```diff
@@ -2,38 +2,38 @@
     <!-- Heading Section -->
     <div class="mb-5 flex items-center justify-between gap-4 max-sm:flex-wrap">
         <div class="grid gap-1.5">
-            <div class="shimmer h-6 w-[151px] pt-1.5"></div>
+            <div class="shimmer h-6 w-36 pt-1.5"></div>
 
-            <div class="shimmer h-4 w-[151px]"></div>
+            <div class="shimmer h-4 w-40"></div>
         </div>
     </div>
 
-    <!-- Main Content Pannel -->
-    <div class="box-shadow flex h-[calc(100vh-179px)] max-w-max flex-col justify-between rounded-md bg-white dark:bg-gray-900">
+    <!-- Main Content Panel -->
+    <div class="box-shadow flex h-[calc(100vh-179px)] max-w-max flex-col justify-between rounded-md">
         <div>
             <!-- Multiple Tabs -->
-            <div class="journal-scroll flex overflow-auto border-b dark:border-gray-800">
-                <div class="flex w-[85px] gap-1 border-b-2 px-4 py-4">
-                    <div class="shimmer h-[18px] w-[110px]"></div>
+            <div class="flex border-b dark:border-gray-800">
+                <div class="flex w-[83px] gap-1 border-b-2 px-4 py-4 dark:border-gray-800">
+                    <div class="shimmer h-[18px] w-28"></div>
                 </div>
     
                 @for ($i = 1; $i < 6; $i++)
-                    <div class="flex w-[153px] gap-1 border-b-2 px-4 py-4">
-                        <div class="shimmer h-[18px] w-[110px]"></div>
+                    <div class="flex w-[152px] gap-1 border-b-2 px-4 py-4 dark:border-gray-800">
+                        <div class="shimmer h-[18px] w-28"></div>
                     </div>    
                 @endfor
             </div>
 
             <!-- Notifications List -->
-            <div class="journal-scroll grid max-h-[calc(100vh-330px)] overflow-auto">
-                @for ($i = 1; $i < 7; $i++)
-                    <div class="flex h-14 items-start gap-1.5 p-4 hover:bg-gray-50 dark:hover:bg-gray-950">
+            <div class="grid max-h-[calc(100vh-330px)]">
+                @for ($i = 1; $i < 6; $i++)
```

Comment:
Here, revert the removed classes and update the foreach loop to 7 lengths for the big screen.

Reply:
There is no fixed ratio for a big screen, so there is no need to fix the length for static shimmer. If we use the scroll class, then the ratio will be greater than 10.